mysql/mydecimal_test.go

```diff
@@ -554,3 +558,87 @@ func (s *testMyDecimalSuite) TestSub(c *C) {
 		c.Assert(string(result), Equals, ca.result)
 	}
 }
+
+func (s *testMyDecimalSuite) TestMul(c *C) {
+	type tcase struct {
+		a      string
+		b      string
+		result string
+		ec     int
+	}
+	cases := []tcase{
+		{"12", "10", "120", 0},
+		{"-123.456", "98765.4321", "-12193185.1853376", 0},
+		{"-123456000000", "98765432100000", "-12193185185337600000000000", 0},
```

Comment:
is there any test case that covers overflow?  which is more than 81 digits a decimal can represent